config/rbac/role.yaml

```diff
@@ -6,6 +6,69 @@ metadata:
   creationTimestamp: null
   name: snoopy-operator-role
 rules:
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - batch
+  resources:
+  - cronjobs
+  - jobs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - pods
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
```

Comment:
Is it not redundant with the declaration above?

Reply:
What exactly is redundant? I couldn't see it. Let me know I can fix it.

Reply:
The "redundancy" would have been in config/samples/rbac_snoopy_operator_dev/role_scc.yaml which is removed in this PR.  unless im missing something there is no redundancy left.